front/src/views/news/news.view.html

```diff
@@ -1,3 +1,25 @@
-<div>    
-  This is a news page.
+<div id = "wrap">    
+  <el-row class = "row">
+    <el-col :span="9">
+      <el-image fit ='cover' :src='newsList[0].previewImage'></el-image>
+    </el-col>
+      <el-col :span="15">
+        <div class = "main-information-wrap">
+          <div class = "title">
+            <h1>{{newsList[0].title}}</h1>
+          </div>
+          <div class = "main-information">
+            <div class = "info-element secondary"><p>{{newsList[0].publishTime}}</p></div>
+            <div class = "info-element secondary"><p>{{newsList[0].countViews}} просмотров</p></div>
+            <div class = "secondary"><p>{{newsList[0].author.firstName}} {{newsList[0].author.lastName}}</p></div>
+            <div class = "secondary"><p>, {{newsList[0].categories |modifyCategories}}</p></div>
+          </div>
+          <div class = "content">{{newsList[0].content}}</div>
+          <div class = "footer">
+            <div><p>{{newsList[0].countCommentaries}} комментариев</p></div>
+            <div><p>Читать полностью</p></div>
+          </div>
+        </div>
+      </el-col>
+  </el-row>
```

Comment:
it wouln't work forcase with more news. atata/ it is not so difficult, just use v-for directive

Reply:
done